providers/catalog.go

```diff
@@ -278,11 +279,40 @@ var catalog = CatalogType{ // nolint:gochecknoglobals
 		},
 	},
 
+	// Klaviyo configuration
+	Klaviyo: {
+		AuthType: Oauth2,
+		BaseURL:  "https://a.klaviyo.com/api",
+		OauthOpts: OauthOpts{
+			GrantType:                 PKCE,
+			AuthURL:                   "https://www.klaviyo.com/oauth/authorize",
+			TokenURL:                  "https://a.klaviyo.com/oauth/token",
+			ExplicitScopesRequired:    true,
+			ExplicitWorkspaceRequired: false,
+			TokenMetadataFields: TokenMetadataFields{
+				ScopesField: "scope",
+			},
+		},
+		Support: Support{
```

Comment:
The `Support` field is for Ampersand to consume & understand if a connector currently supports the given functionalities, rather than the provider itself, so let's set all of these as false. At a later point in time, post integration testing, we can promote `proxy` to true.